src/components/common/CustomSwitch.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import PropTypes from 'prop-types';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles((theme) => ({
+  wrapper: {
+    margin: theme.spacing(2, 0),
+    display: 'flex',
+    justifyContent: 'space-between',
+    width: '55%',
```

Comment:
I don't really like this floating switch. Maybe we can display it really next to the label and later introduce a table to align every options / switches?